tools/c7n_azure/c7n_azure/constants.py

```diff
@@ -25,6 +25,19 @@
 FUNCTION_PACKAGE_SAS_EXPIRY_DAYS = 365 * 10  # 10 years
 FUNCTION_AUTOSCALE_NAME = 'cloud_custodian_default'
 
+"""
+Azure Container Host
+"""
+CONTAINER_EVENT_TRIGGER_MODE = 'container-event'
+CONTAINER_TIME_TRIGGER_MODE = 'container-periodic'
+ENV_CONTAINER_EVENT_QUEUE_ID = 'AZURE_EVENT_QUEUE_RESOURCE_ID'
+ENV_CONTAINER_EVENT_QUEUE_NAME = 'AZURE_EVENT_QUEUE_NAME'
+ENV_CONTAINER_POLICY_STORAGE = 'AZURE_CONTAINER_STORAGE'
+ENV_CONTAINER_OPTION_LOG_GROUP = 'AZURE_CONTAINER_LOG_GROUP'
```

Comment:
ideally, this should be using click and taking custodian standard cli parameters rather than introducing a whole new azure specific set of environment variables.